src/connector/src/sink/iceberg/mod.rs

```diff
@@ -83,7 +80,10 @@ pub struct IcebergConfig {
     pub table_name: String, // Full name of table, must include schema name
 
     #[serde(rename = "database.name")]
-    pub database_name: String, // Database name of table
+    pub database_name: Option<String>, // Database name of table
+
+    #[serde(rename = "catalog.name")]
+    pub catalog_name: String, // Catalog name.
```

Comment:
Is it a breaking change or user-facing? It looks like we need to specify the catalog name for the iceberg sink and source.

Reply:
It's user facing. This field is useless for storage catalog, but must be set for others such as jdbc, hive catalog.

Reply:
If that is the case, should it be optional? Otherwise, users must need to specify the storage catalog in iceberg sink or iceberg source.

Reply:
Seems reasonable.